app/main.py

```diff
@@ -2,0 +1,25 @@
+import os
+
+
+def move_file(command: str) -> None:
+    instructions = command.split()
+    _, source, destination = instructions
+
+    if len(instructions) != 3 or instructions[0] != "mv":
```

Comment:
The input validation logic is correct, but ensure that the command is split correctly and that there are exactly three parts. This aligns with the checklist point: 'Make sure you check that the input string contains 3 elements separated by space, and has `mv` on a first place.'